dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderFactoryImpl.java

```diff
@@ -101,7 +101,7 @@ protected Folder find(String folderInode) throws DotDataException {
 		if (folder == null) {
 			try{
 			     DotConnect dc    = new DotConnect()
-			             .setSQL("SELECT folder.*, folder_1_.* from folder folder, inode folder_1_ where folder.inode = ? or folder.identifier=?")
+			             .setSQL("SELECT * from folder where inode = ? or identifier=?")

```

Comment:
We need the inode information as well to build the folder object, no?

Reply:
No, the Folder transformer reads everything from the Folder table.. the inode part is just the inode string from the folder table